tasmoadmin/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN \
         php7=7.3.11-r0 \
     \
     && apk add --no-cache --virtual .build-dependencies \
-        git=2.22.0-r0 \
+        git=2.22.2-r0 \
```

Comment:
This should not be in this PR.

Reply:
The docker image failed to build without it...so 🤷‍♂